index.html

```diff
@@ -8,51 +8,232 @@
-              or website Look through some of my work and experience! If you
-              like what you see and have a project you need coded, don’t
-              hestiate to contact me.
+      </header>
+      <section class="intro">
+        <h1 class="title">I’m Mahdia, Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <div class="social-media">
+          <h5>LETS CONNECT</h5>
+          <ul>
+            <li>
+              <a href=""><img src="twitter.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="linked.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="meduimsvg.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="github.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="angellist.svg" alt="" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <div class="cards">
+          <img src="Snapshoot.svg" alt="" />
+          <div class="projects">
+            <div class="project-section">
+              <h1 class="project-title">Tonic</h1>
+              <div class="extra-info">
+                <p class="proj-name">CANOPY</p>
+                <img src="Circle.svg" alt="" />
+                <p class="job-title">Back End dev</p>
+                <img src="Circle.svg" alt="" />
+                <p class="job-title">2015</p>
+              </div>
+            </div>
+            <p class="details">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
             </p>
-            <a href="">LET’S CONNECT</a>
+            <ul class="lang-tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="link">
+              <a href="">See Project</a>
+            </div>
           </div>
-        </section>
-      </div>
-    </div>
-    <div class="img">
-      <ul>
-        <li>
-          <a href="#"> <img src="twitter.svg" class="twitter" alt="" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="linked.svg" class="linkedIn" alt="" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="Vector-(1).svg" class="meduim" alt="" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="Vector.svg" class="angellist" alt="" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="angellist.svg" class="angellist" alt="" /></a>
-        </li>
-      </ul>
+        </div>
+      </section>
+            <section class="works">
+        <div class="cards">
```

Comment:
- [ ] Kindly make sure all the cards are wrapped in one section tag "works". There is no need to add different section tags for each card.